.rubocop.yml

```diff
@@ -0,0 +1,19 @@
+require: rubocop-rails
+
+AllCops:
+  NewCops: enable
+  
+Rails:
+  Enabled: true
+
+Style/FrozenStringLiteralComment:
+  Enabled: false
```

Comment:
What is the point disabling this rule? It is good practice to have string literals frozen.